### PR TITLE
testing/py-packaging: modernize APKBUILD

### PR DESCRIPTION
**Pull request by rgdoliveira**
Changes in APKBUILD to make it complaint with python model

### CODE REVIEW

File: testing/py-packaging/APKBUILD [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
 pkgdesc="Core utilities for Python packages"
 url="https://pypi.python.org/pypi/packaging"
-arch="all"
+arch="noarch"
 license="APACHE or BSD"
 subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
 makedepends="python3-dev python2-dev py-setuptools"
-source="https://pypi.python.org/packages/c6/70/bb32913de251017e266c5114d0a645f262fb10ebc9bf6de894966d124e35/packaging-$pkgver.tar.gz"
-builddir="$srcdir/packaging-$pkgver"
+source="https://pypi.python.org/packages/c6/70/bb32913de251017e266c5114d0a645f262fb10ebc9bf6de894966d124e35/$_pkgname-$pkgver.tar.gz"
```
Comment (misery) [inline testing/py-packaging/APKBUILD]: You could use this:

https://files.pythonhosted.org/packages/source/p/$_pkgname/$_pkgname-$pkgver.tar.gz

**Comment by rgdoliveira** [comment]
@misery thanks, just applied your suggestion

**Comment by jirutka** [comment]
There are also missing dependencies. According to project’s [setup.py](https://github.com/pypa/packaging/blob/16.8/setup.py#L53), this module needs pyparsing and six.

**Comment by rgdoliveira** [comment]
thanks for the review @jirutka, I applied your suggestion.

**Comment by jirutka** [comment]
You have to also add `depends="${depends//py-/py2-}"` and `depends="${depends//py-/py3-}"` to `_py2` and `_py3()` split function, respectively.

**Pull request closed without merging**